### PR TITLE
[FIX] point_of_sale: fix some issues for POS printing

### PR DESCRIPTION
**Pull request by rfr-odoo**
Like commit odoo/odoo@1c5c762
on some device (like Android Mobile) the black background is printed.
We change the background to white to avoid this.

opw-1940434

----

On mobile, the "document.execCommand('print')" call returns "true"
but do nothing, so it doesn't exec the "window.print()".

As the "print" command is deprecated, "window.print()" can be safely called directly.

opw-2486818

### CODE REVIEW

File: addons/point_of_sale/static/src/js/Misc/AbstractReceiptScreen.js [hunk 1]
```diff
@@ -44,8 +45,12 @@ odoo.define('point_of_sale.AbstractReceiptScreen', function (require) {
          */
         async _printWeb() {
             try {
-                const isPrinted = document.execCommand('print', false, null);
-                if (!isPrinted) window.print();
+                if (methods.switchAccount) {
+                    window.print();
+                } else {
+                    const isPrinted = document.execCommand('print', false, null);
```
Comment (adr-odoo) [inline addons/point_of_sale/static/src/js/Misc/AbstractReceiptScreen.js:51]: @caburj  Do you remember in which cases window.print doesn't work? I already check the link but I also see that we only do 'document.execCommand('print', false, null);' for Safari in Odoo see https://github.com/odoo/odoo/blob/4629a4fb0eacedb024562ef5542011d2e0b2c1f4/addons/point_of_sale/static/src/js/screens.js#L1490.

Comment (caburj) [reply addons/point_of_sale/static/src/js/Misc/AbstractReceiptScreen.js:51]: In the stackoverflow link, I think I accepted the answer without any proper investigation. Therefore, I don't remember anything about which browser/version the `window.print` doesn't work. Basically, I assumed that for all, `document.execCommand('print', false, null)` should be called first before `window.print`, but based on this PR, it turns out that result of `document.execCommand('print', false, null)` is not reliable on mobile.

I checked this: https://caniuse.com/?search=window.print and it appears that window.print seems to be safe.

**Comment by pparidans** [comment]
robodoo rebase-ff

**Comment by pparidans** [comment]
robodoo r+

**Review by pparidans — APPROVED** [review]

**Pull request closed without merging**